### ISSUE TITLE
Some labels are invisible in List Controller due to inline "display: none;" style

### ISSUE THREAD

**Issue by goldmont** [wintercms/winter#88]
Hi there,

after moving to WinterCMS, controllers lists got broken. As you can see from the attached image, the title column content is invisible because an inline "display: none;" style is attached somehow. I didn't managed to solve this issue and I don't understand why this happens only on that column. Hope there's a solution. Thank you for your efforts.

```yaml
columns:
    title:
        label: Title
        type: text
    status:
        label: Completed
        searchable: false
        sortable: true
        type: switch
        width: 10%

```

Result:
![image](https://user-images.githubusercontent.com/6855692/115130317-329bbf00-9fef-11eb-98d0-15d6b4f8a981.png)

**Comment by LukeTowers** [comment]
@goldmont this was fixed in https://github.com/wintercms/winter/commit/2bb5d44eb564dc5ca1371258102b175e1778c159 and will be available in v1.1.3. If you would like to get access to the fix now, switch your composer dependencies to use dev-develop instead.

**Comment by goldmont** [comment]
Thank you man
